prepare_submodule.sh

```diff
@@ -22,3 +22,4 @@ set -e
 git submodule update --init --depth 1
 cd js_mse_eme
 git apply ../patches/0001_js_mse_eme_mvt.patch
+git apply ../patches/0002_js_mse_eme_mvt.patch
```

Comment:
patch name should be like 0002_js_mse_eme_mvt_AudioTagChangeFor_FMP4_FMP3_stream.patch